astropy/units/tests/test_quantity_non_ufuncs.py

```diff
@@ -1392,6 +1392,27 @@ def test_histogram(self):
         with pytest.raises(u.UnitsError):
             np.histogram(x.value, [125, 200] * u.s)
 
+    @classmethod
+    def _range_value(cls, range, unit):
+        if isinstance(range, u.Quantity):
+            return range.to(unit).value
+        else:
+            return [cls._range_value(r, unit) for r in range]
+
+    @pytest.mark.parametrize("range", [[2 * u.m, 500 * u.cm], [2, 5] * u.m])
+    @needs_array_function
+    def test_histogram_range(self, range):
+        self.check(
+            np.histogram,
+            self.x,
+            range=range,
+            value_args=[
+                self.x.value,
```

Comment:
I think we can get black to put this on a single line by removing the trailing `,` here.